### DISCUSSION TITLE
Bun:sqlite - Unable to create multiple tables with the same `run()`?

### DISCUSSION THREAD

**Discussion by charnould** [oven-sh/bun#4003] [Q&A]
Considering simple function below:

```
export const init_db = async () => {
  const sql = await Bun.file('helpers/migrate/v.0.0.sql').text()
  const events_db = new Database(`./datastore/events.db`)
  events_db.query(sql).run()
  return
}
```

...and a simplified `v.0.0.sql` containing:  
(It also contains `create trigger` but there are currently commented out)

```
-- Create events table
CREATE TABLE
  IF NOT EXISTS events (
    id TEXT PRIMARY KEY,
    support_phone TEXT DEFAULT NULL
  );

-- Create calendar table
CREATE TABLE
  IF NOT EXISTS calendar (
    id TEXT NOT NULL,
    start TEXT NOT NULL,
    end TEXT NOT NULL,
    adult_fee REAL NOT NULL,
    child_fee REAL NOT NULL
  );

-- Create linked_to table
CREATE TABLE
  IF NOT EXISTS linked_to (id TEXT NOT NULL, domain TEXT NOT NULL);
```

**Result**
`events` table is created, but not `calendar` and `linked`, and no error is thrown.  
Is there a problem with my code or with `Bun:sqlite` ? Thanks.

**Comment by Jarred-Sumner** [comment]
You can verify whether or not this is a problem with bun:sqlite or sqlite via the `sqlite` CLI. It is unlikely to be a problem with bun:sqlite

**Reply by charnould** [reply]
My code runs and works. I do load my `.sql` file using `sqlite3` cli.  
Consequence is, I need to include it into my `docker` image and it it weighs.

Apparently, SQLite does not allow it (?):
- https://stackoverflow.com/questions/2928184/only-first-table-in-create-table-statement-being-created
- https://stackoverflow.com/questions/27669987/sqlite3-create-multiple-tables-in-one-statement

Sad...!
